docs/understanding-your-pipeline/canonical-event/index.md

```diff
@@ -36,274 +42,490 @@ The Snowplow data table is designed to be immutable: the data in each line shoul
+| `refr_urlhost`                                            | text      | Referrer host                                                              | No    | 'www.bing.com'                                                 | [Referrer Parser Enrichment](/docs/enriching-your-data/available-enrichments/referrer-parser-enrichment/index.md)                                                                                                                                                                    |
+| `refr_urlport`                                            | int       | Referrer port                                                              | No    | 80                                                             | [Referrer Parser Enrichment](/docs/enriching-your-data/available-enrichments/referrer-parser-enrichment/index.md)                                                                                                                                                                    |
+| `refr_urlpath`                                            | text      | Referrer page path                                                         | No    | '/images/search'                                               | [Referrer Parser Enrichment](/docs/enriching-your-data/available-enrichments/referrer-parser-enrichment/index.md)                                                                                                                                                                    |
+| `refr_urlquery`                                           | text      | Referrer URL querystring                                                   | No    | 'q=psychic+oracle+cards'                                       | [Referrer Parser Enrichment](/docs/enriching-your-data/available-enrichments/referrer-parser-enrichment/index.md)                                                                                                                                                                    |
+| `refr_urlfragment`                                        | text      | Referrer URL fragment                                                      | No    |                                                                | [Referrer Parser Enrichment](/docs/enriching-your-data/available-enrichments/referrer-parser-enrichment/index.md)                                                                                                                                                                    |
```

Comment:
As above, these are just "Default Enrichment (`referer`)", only medium/source/term are populated by the enrichment.